apps/owmweather/lib.js

```diff
@@ -1,19 +1,20 @@
 function parseWeather(response) {
   let owmData = JSON.parse(response);
 
-  let isOwmData = owmData.coord && owmData.weather && owmData.main;
+  let isOwmData = (owmData.lat && owmData.lon) && owmData.current.weather && owmData.current;
```

Comment:
Now we're going deeper, do we want to guard against `.current` missing here? Perhaps:
```suggestion
  let isOwmData = false;
  try {
    isOwmData = (owmData.lat && owmData.lon) && owmData.current.weather && owmData.current;
  } catch (_e) {}
```

Reply:
Implemented.